### PR TITLE
Issue # 1065| Adding a block to the slider documentation

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/merchdocs/issues/1065

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/cms/page-builder-media-slider.html#complex-content-in-the-settings

whatsnew
Updated [Media - Slider](https://docs.magento.com/user-guide/cms/page-builder-media-slider.html) page with new section for adding a widget to a slide.

### CODE REVIEW

File: src/cms/page-builder-media-slider.md [hunk 1]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
 
 1. When complete, click <span class="btn">Save</span> in the upper-right corner to close the _Edit Slide_ page.
 
+### Add widget on a slide
+
+You can add any [widgets types]({% link cms/widget-types.md %}) to your slide in a page bilder stage with the following steps:
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:123]:
```suggestion
You can add any [widgets type]({% link cms/widget-types.md %}) to your slide in a Page Builder stage using the following steps:
```

File: src/cms/page-builder-media-slider.md [hunk 2]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+1. [Create the widget] ({% link cms/widget-create.md %}) you want to see on a slide.
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:125]:
```suggestion
1. [Create the widget]({% link cms/widget-create.md %}) you want to see on a slide.
```

File: src/cms/page-builder-media-slider.md [hunk 3]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+1. Open the page, block, or dynamic block where you want to place the slider and expand the **Content** section.
+
+1. In the Page Builder panel, expand **Media** and drag a **Slider** placeholder to a row, column, or tab on the stage.
+
+1. Click in the slider container to display the [text editor]({% link cms/editor.md %}).
+
+1. In the text editor click the **Insert Widget** ![]({% link images/images-ee/insert-widget-icon.png  %})).
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:133]: I would combine these to a single step and the widget icon already exists, so let's remove the new image file.

```suggestion
1. Click in the slider container to display the [text editor]({% link cms/editor.md %}) toolbar and click the _Insert Widget_ ![]({% link images/images/editor4-toolbar/btn-insert-widget.png  %})) icon.
```

File: src/cms/page-builder-media-slider.md [hunk 4]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+1. Select the **Widget Type** you necessary.
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:135]:
```suggestion
1. Select the **Widget Type** you need.
```

File: src/cms/page-builder-media-slider.md [hunk 5]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+1. Specify the necessary settings (_Note:_ they are different for each widget).
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:137]:
```suggestion
1. Specify the settings, which are different depending on the type of widget
```

File: src/cms/page-builder-media-slider.md [hunk 6]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+   ![]({% link images/images-ee/insert-widget-to-slide-page.png %}){: .zoom}
+   _Example of inserting widget on the slide_
+
+1. When complete, click <span class="btn">Insert Widget</span> in the upper-right corner.
+
+1. Modify the other settings, as needed.
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:144]:
```suggestion
1. Modify the other settings as needed.
```

File: src/cms/page-builder-media-slider.md [hunk 7]
```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
...
+
+1. When complete, click <span class="btn">Save</span> in the upper-right corner.
+
+   ![]({% link images/images-ee/inserting-widget-on-slide.png %}){: .zoom}
+   _Example of inserting widged on the slide_
```
Comment (leslietilling) [inline src/cms/page-builder-media-slider.md:149]:
```suggestion
   _Example of inserted widget on the slide_
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
A few edits and replace icon with existing file.

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing :) 
I have implemented your suggestions, could you please approve this PR?

Thank you!

File: src/cms/page-builder-media-slider.md (hunk 7 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

File: src/cms/page-builder-media-slider.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-slider.md]: Done ✅

**Comment by leslietilling** [comment]
@EvheniiaKolkhiieva  We also need to remove this duplicate image file `src/images/images-ee/insert-widget-icon.png `

**Comment by EvheniiaKolkhiieva** [comment]
Oh yes, sorry @leslietilling :)
I have removed this file
Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**